plugins/woocommerce/tests/e2e-pw/fixtures/install-wc.setup.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Internal dependencies
+ */
+import { test as setup } from './fixtures';
+
+async function deactivateWooCommerce( wcbtApi ) {
+	try {
+		await wcbtApi.fetch(
+			'/wp-json/wc-admin-test-helper/live-branches/deactivate/v1',
+			{ method: 'GET' }
+		);
+		console.log( 'WC deactivated.' );
+	} catch ( err ) {
+		console.error( 'Error deactivating WooCommerce:', err );
+	}
+}
+
+setup( 'Install WC using WC Beta Tester', async ( { wcbtApi, wpApi } ) => {
+	setup.skip(
+		! process.env.INSTALL_WC,
+		'Skipping installing WC using WC Beta Tester; INSTALL_WC not found.'
+	);
+	console.log( 'INSTALL_WC is enabled. Running installation script...' );
+
+	// Check if WooCommerce is activated and its version
+	const response = await wpApi.get( './wp-json/wp/v2/plugins', {
+		data: { status: 'active' },
+	} );
+	const plugins = await response.json();
+	const activatedWcVersion = plugins.find(
+		( plugin ) => plugin.name === 'WooCommerce'
+	)?.version;
+
+	if ( activatedWcVersion ) {
+		console.log(
+			`WooCommerce is activated. Version: ${ activatedWcVersion }`
+		);
+	} else {
+		console.log( 'WooCommerce is not activated.' );
+	}
+
+	const wcVersion = process.env.WC_VERSION || 'latest';
+	let resolvedVersion = '';
+
+	// Install WC
+	if ( wcVersion === 'latest' ) {
+		const latestResponse = await wcbtApi.fetch(
+			'/wp-json/wc-admin-test-helper/live-branches/install/latest/v1',
+			{
+				method: 'POST',
+				data: { include_pre_releases: true },
+			}
+		);
+
+		if ( ! latestResponse.ok() ) {
+			throw new Error(
+				`Failed to install latest WC: ${ latestResponse.status() } ${ await latestResponse.text() }`
+			);
+		}
+
+		resolvedVersion = ( await latestResponse.json() )?.version || '';
+
+		setup.skip(
+			resolvedVersion === activatedWcVersion,
+			'Skip installing WC: The latest version is already installed and activated.'
+		);
```

Comment:
I would no use `setup.skip` at this point, it already executed half of the setup and leads to confusion. The step was marked as skipped while the log was telling me `INSTALL_WC is enabled. Running installation script...`

```suggestion
		if(resolvedVersion === activatedWcVersion) {
			console.log('Required version is already installed and active.');
			return;
		}
```

Reply:
You're right. Logging was confusing indeed. I removed `setup.skip` from two places and improved logging here [c15600c](https://github.com/woocommerce/woocommerce/pull/55063/commits/c15600c9d45e2c1ff89521e4bc74fb5b4eda6103)